File: aa/DD/PP/BB/CC/aa.py
#!/usr/bin/python3

class Hello():
	def __init__(self,name):
		self.name = name
	
	def ADD(self,a,b):
		sum = a+b
		return sum

	def Update(self,pp):
		self.name = pp

	def Print(self):
		print (self.name)

aa = Hello("srinu")
aa.Print()
print (aa.name)

print (aa.ADD(10,20))

aa.Update("vas")

print ("update the name is ",aa.name )

aa.Print()

File: aa/nn/aa.pl
#!/usr/bin/perl -w
use strict;

package AB;


sub hello {
		my $aa = shift;
		print "$aa\n";
		sub good {
				my $bb = shift;
				print "$bb\n";
		}
}





&AB::hello(100);
&AB::good(10);


hello(101);
good(11);



package CD;

our $name = "SRINU";			
